portal-impl/src/com/liferay/portal/language/LanguageImpl.java

```diff
@@ -1966,9 +1972,9 @@ private CompanyLocalesBag(long companyId) {
 			}
 
 			_availableLocales = Collections.unmodifiableSet(
-				new HashSet<>(_languageIdLocalesMap.values()));
+				new LinkedHashSet<>(_languageIdLocalesMap.values()));
 
-			Set<Locale> supportedLocalesSet = new HashSet<>(
+			Set<Locale> supportedLocalesSet = new LinkedHashSet<>(
```

Comment:
why do we need to change this var. From my understanding, only available language is ordered in UI.

Reply:
For this part, I do this because I think it may be used, If it's not necessary, I can change them back.

Reply:
I want to keep the original because only available language is ordered in UI.